alpha/lib/model/UserLoginData.php

```diff
@@ -344,6 +344,19 @@ public function getInvalidPasswordStructureMessage(){
 		return $invalidPasswordStructureMessage; 
 	}
 	
+	public function isCommonPassword($password)
+	{
+		$commonPasswordsMap = kConf::getMap('commonPasswords');
```

Comment:
add map name to kConfMapNames.php and take it from there
map name should be lower case with underscores